### ISSUE TITLE
Use scikit-learn's new balanced accuracy metric

### ISSUE THREAD

**Issue by mfeurer** [automl/auto-sklearn#700]
Which is described in their [docs](https://scikit-learn.org/stable/modules/model_evaluation.html#balanced-accuracy-score).

**Comment by mfeurer** [comment]
Closed via #814.
